mojo/stdlib/stdlib/python/_cpython.mojo

```diff
@@ -1338,14 +1355,17 @@ struct CPython(Defaultable, Movable):
     var _PyType_GenericAlloc: PyType_GenericAlloc.type
     var _PyType_GetName: PyType_GetName.type
     var _PyType_FromSpec: PyType_FromSpec.type
+    var _PyType_GetFlags: PyType_GetFlags.type
     # The None Object
     var _Py_None: PyObjectPtr
     # Integer Objects
     var _PyLong_FromSsize_t: PyLong_FromSsize_t.type
     var _PyLong_FromSize_t: PyLong_FromSize_t.type
     var _PyLong_AsSsize_t: PyLong_AsSsize_t.type
+    var _PyLong_Type: PyTypeObjectPtr
```

Comment:
Please move this before the other functions (of the same section), since we order everything (fields, type def, methods) strictly following Python's documentation pages. Do the same for the other fields you added. It might feel a bit rigid, but I'd like to keep everything very organised and delegate the "boring" or potentially bikeshedding-prone details to Python.